docs/fastdds/dds_layer/core/entity/entity.rst

```diff
@@ -89,6 +89,19 @@ Then they can link these listeners to each entity, either during their creation
 |DataReader::set_listener-api|).
 The listener interfaces that each entity type and their callbacks are explained in the documentation
 for each entity type.
+In the case in which the listeners of different entities have the same callback defined, the order in which each
+callback is executed is dictated by the inheritance from parent to child, i.e. from the lowest level entity to the
+highest. 
```

Comment:
This needs rewording. Is not only when the listener has the callback defined, but that any event is handled by the lowest-level entity that:
- has a not-null listener attached
- and the specific callback is enabled on the StatusMask
Also, `the order in which each callback is executed` suggests that all listeners are triggered one after the other.